### PR TITLE
Fixes #17069 - foreman-debug counts size instead lines

### PR DESCRIPTION
**Pull request by pmoravec**
.. instead of tailing all (log)files

Update calling add_files (modified in foreman-debug).

Signed-off-by: Pavel Moravec pmoravec@redhat.com

### CODE REVIEW

**Comment by domcleal** [comment]
Administrative note: this ticket is not from the Smart Proxy project, it's in Katello. Please file a ticket describing the changes in Smart Proxy. It also appears to depend on changes from https://github.com/theforeman/foreman/pull/3881.

**Comment by dmitri-d** [comment]
Please see the administrative note above.

File: extra/foreman-debug-proxy [hunk 1]
```diff
@@ -21,15 +25,18 @@ add_files /etc/xinetd.d/tftp
...
+add_files /etc/named.conf
+add_files /var/log/named.log
+for SUBDIR in /var/named /var/named/{data,dynamic,slaves}; do
+  add_files $SUBDIR/
+done
 
 # Puppet master and CA
 add_files /var/lib/puppet/ssl/certs/$(hostname -f).pem /var/lib/puppet/ssl/certs/ca.pem
-add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/* /var/log/puppet/*.log*
+add_files /etc/puppet/*.conf /etc/puppet/rack/* /etc/puppet/manifests/*
```
Comment (dmitri-d) [inline extra/foreman-debug-proxy:34]: These are locations for older versions of puppet (pre 4.0). Current versions use `/etc/puppetlabs/*`.

File: extra/foreman-debug-proxy [hunk 2]
```diff
@@ -21,15 +25,18 @@ add_files /etc/xinetd.d/tftp
...
+add_files /var/log/puppet/*.log*
```
Comment (dmitri-d) [inline extra/foreman-debug-proxy:35]: Similarly, `/var/log/puppetlabs` and `var/log/puppetserver` directories are used by current versions of puppet.

File: extra/foreman-debug-proxy [hunk 3]
```diff
@@ -21,15 +25,18 @@ add_files /etc/xinetd.d/tftp
...
 add_files /etc/puppet/ssl/ca/inventory.txt /var/lib/puppet/ssl/ca/inventory.txt
 add_cmd "find /etc/puppet/modules -exec ls -ld {} +" "puppet_manifests_tree"
```
Comment (dmitri-d) [inline extra/foreman-debug-proxy:37]: Same issues as above.

**Review by dmitri-d — DISMISSED** [review]

File: extra/foreman-debug-proxy (hunk 1 above)
Comment (pmoravec) [reply extra/foreman-debug-proxy:34]: Can't upstream smart-proxy run with older puppet as well?

Anyway, I suggest filing new issue for this. Here in the PR I just reflect the planned change made to foreman-debug. My change does not bring a regression (I hope) and is independent on the puppet version / puppetlabs dir.

**Comment by dmitri-d** [comment]
Waiting for https://github.com/theforeman/foreman/pull/3881 to get merged.

**Comment by lzap** [comment]
Thanks, created ticket for the puppet paths: http://projects.theforeman.org/issues/17261

**Pull request merged**